cmds/core/ip/link_linux.go

```diff
@@ -160,6 +160,9 @@ func (cmd *cmd) setLinkMTU(iface netlink.Link) error {
 	return cmd.handle.LinkSetMTU(iface, mtu)
 }
 
+// TODO: get author review
+//
+//nolint:unused
```

Comment:
just remove the parameter, right?

Reply:
The problem is that the function is not called, and I wanted the authors to review that to check why this is the case. @RiSKeD authored this so he might want to comment on that before we remove it.

Reply:
Yes, this is fine for now. I will do a PR with a fix using it. Thanks for the headsup

Reply:
The function is now being used, so you can remove the linter tag